src/main/java/team4/footwithme/team/api/TeamMemberApi.java

```diff
@@ -1,11 +1,49 @@
 package team4.footwithme.team.api;
 
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import team4.footwithme.global.api.ApiResponse;
+import team4.footwithme.team.api.request.TeamMemberRequest;
+import team4.footwithme.team.service.TeamMemberService;
+import team4.footwithme.team.service.response.TeamResponse;
+
+import java.util.List;
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping
+@RequestMapping("/api/v1/team_member")
 public class TeamMemberApi {
+    private final TeamMemberService teamMemberService;
+
+    /**
+     * 팀 멤버 추가
+     */
+    @PostMapping("/{teamId}/members")
+    public ApiResponse<List<TeamResponse>> addTeamMembers(@PathVariable Long teamId, @RequestBody TeamMemberRequest request){
+        List<TeamResponse> members = teamMemberService.addTeamMembers(teamId, request.toServiceRequest());
+        return ApiResponse.ok(members);
```

Comment:
```suggestion
        return ApiResponse.ok(teamMemberService.addTeamMembers(teamId, request.toServiceRequest()));
```

저는 이렇게 씁니다

이건 취향차이인거같아요

Reply:
불필요한 리스트를 계속 생성하는 것 같아서 맘에 안들긴했습니다. 제안해주신대로 수정하는게 좋을것 같아요!